sirepo/job_supervisor.py

```diff
@@ -299,13 +299,18 @@ def _purge_sim(db_file):
             if d.lastUpdateTime > _too_old:
                 return
             if d.status == job.JOB_RUN_PURGED:
+                cls._purged_jids_cache.add(db_file.purebasename)
```

Comment:
Move this one line up, and delete line 307.

Reply:
https://github.com/radiasoft/sirepo/pull/2521/commits/bf9c680f975b12ee38bfa0b0e494fb3e3abe90f7